### PR TITLE
Ported command 'ee secure'

### PR DESCRIPTION
**Pull request by bnap00**
As `ee secure --port` and `ee secure --ip` was not at all implemented,
I ported command `ee secure --port` and `ee secure --ip` which is linked down below,
>https://easyengine.io/docs/commands/secure/

### CODE REVIEW

**Comment by bnap00** [comment]
As `ee secure port` was not at all implemented,
I ported command `ee secure port` which is linked down below,
>https://easyengine.io/docs/commands/secure/

File: php/commands/secure.php [hunk 1]
```diff
@@ -36,11 +36,60 @@ public function secure_port(){
...
+		EE::debug("Adding changes to git");
+		EE_Git::add("/etc/nginx","Adding changed port to git");
+		EE::debug("Restarting nginx");
+		EE_Service::restart_service("nginx");
 	}
 
 	public static function secure_ip(){
 		//***This function Secures IP***
 		//todo:
+		$newiplist = array();
```
Comment (rahulsprajapati) [inline php/commands/secure.php]: @bnap00 
I think you are not using this var can you please remove this one if not ?

File: php/commands/secure.php [hunk 2]
```diff
@@ -36,11 +36,59 @@ public function secure_port(){
...
+		
+		while(!is_numeric($port)){
+			EE::debug("Not a valid port");
+			EE::error("Please enter valid port number:");
+			$port = EE::input_value("EasyEngine admin port [22222]:");
+		}
+		EE::debug("Checking distro of the machine");
+		$distro_name = EE_OS::ee_platform_distro();
+		EE::debug("$distro_name found");
+		if("ubuntu" == $distro_name) {
```
Comment (mbtamuli) [inline php/commands/secure.php]: The _ubuntu_ and _debian_ distro seem to have same settings for this config. Can be put in one **if** block.

File: php/commands/secure.php [hunk 3]
```diff
@@ -36,11 +36,59 @@ public function secure_port(){
...
+			EE::debug("Changing nginx configuration");
+			EE::exec_cmd("sed -i \"s/listen.*/listen $port default_server ssl http2;/\" /etc/nginx/sites-available/22222");
+		} else if ( "debian" == $distro_name){
```
Comment (mbtamuli) [inline php/commands/secure.php]: Isn't this the same for Ubuntu and Debian? Can't it be put together in the same condition?
 @bnap00 @rahulsprajapati

**Pull request merged**